### PR TITLE
added morse code to the funbox

### PR DESCRIPTION
**Pull request by epicjoanna**
### Description
made by @[DastardlySky](https://github.com/DastardlySky) and i

this PR adds a morse code mode to the funbox, where words are converted to corresponding morse symbols, and "/" in place of spaces between letters/words. 

these changes were made by implementing a function in funbox.ts that includes a dictionary holding key-value pairs of characters and their morse symbol, followed by a conditional loop that iterates through each word and translates it by getting the value (morse symbol) from its key (character).

 
image
image

### CODE REVIEW

File: frontend/src/ts/test/funbox/funbox-list.ts [hunk 1]
```diff
@@ -224,6 +224,11 @@ const list: MonkeyTypes.FunboxMetadata[] = [
     info: "Words are generated according to Zipf's law. (not all languages will produce Zipfy results, use with caution)",
     properties: ["changesWordsFrequency"],
   },
+  {
+    name: "morse",
+    info: "-.../././.--./ -.../---/---/.--./-.-.--/ ",
```
Comment (Miodec) [inline frontend/src/ts/test/funbox/funbox-list.ts:229]: Should this mean something? 
image

File: frontend/src/ts/test/funbox/funbox.ts [hunk 2]
```diff
@@ -723,3 +723,71 @@ export async function rememberSettings(): Promise<void> {
...
+      "!": "-.-.--",
+      "@": ".--.-.",
+    };
+
+    let morseWord = "";
+
+    for (let i = 0; i < word.length; i++) {
+      morseWord += morseCode[word.toLowerCase()[i]] + "/";
+    }
+    return morseWord;
```
Comment (Miodec) [inline frontend/src/ts/test/funbox/funbox.ts:791]: Please move this code to `misc.ts`, `convertToMorse`

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/src/ts/test/funbox/funbox-list.ts (hunk 1 above)
Comment (DastardlySky) [reply frontend/src/ts/test/funbox/funbox-list.ts:229]: yep, it means "beepboop!" in morse code.

In morse code, words and letters are sometimes delimited by "/". If I add spaces using the same site as above, I get this:

image

or if I put in the orginal input on https://morsecode.world/international/translator.html:

image

Comment (Miodec) [reply frontend/src/ts/test/funbox/funbox-list.ts:229]: Alright

File: frontend/src/ts/test/funbox/funbox.ts (hunk 2 above)
Comment (epicjoanna) [reply frontend/src/ts/test/funbox/funbox.ts]: added bulk of this into `misc.ts`

**Comment by Miodec** [comment]
Thanks

**Comment by Miodec** [comment]
Oh yeah, also, in the future please discuss features before adding them :)

**Pull request merged**